src/graphql.rs

```diff
@@ -542,42 +538,24 @@ where
     A: ObjectType,
     NodesField: ConnectionNameType,
 {
-    match (
-        first.is_some(),
-        last.is_some(),
-        before.is_some(),
-        after.is_some(),
-    ) {
-        (true, true, _, _) => return Err("cannot provide both `first` and `last`".into()),
-        (_, _, true, true) => return Err("cannot provide both `before` and `after`".into()),
-        (true, _, true, _) => return Err("cannot provide both `first` and `before`".into()),
-        (_, true, _, true) => return Err("cannot provide both `last` and `after`".into()),
-        (false, false, false, _) => {
-            first = Some(DEFAULT_CONNECTION_SIZE);
-        }
-        (false, false, true, false) => {
-            last = Some(DEFAULT_CONNECTION_SIZE);
-        }
-        _ => {}
-    }
-
-    let after = if let Some(after) = after {
-        Some(decode_cursor(&after).ok_or("invalid cursor `after`")?)
-    } else {
-        None
-    };
-    let before = if let Some(before) = before {
-        Some(decode_cursor(&before).ok_or("invalid cursor `before`")?)
-    } else {
-        None
-    };
-
```

Comment:
Why is this part not needed any longer?

Reply:
This `load` function did not apply to some GraphQL APIs that had pagination therefore this validation and processing did not apply to some while it did to others. To resolve this case, we made a function that could be used for all APIs before calling `query` so that it could adhere to this pagination practice (e.g. `nodeStatusList` did not have this check applied but `customerList` did previously).